pom.xml

```diff
@@ -6,8 +6,9 @@
     <artifactId>soulwing-parent</artifactId>
     <version>13</version>
   </parent>
+  <groupId>org.soulwing</groupId>
   <artifactId>wildfly-jgroups-aws-ping</artifactId>
-  <version>1.0.5-SNAPSHOT</version>
+  <version>1.0.5-VMINTERNAL</version>
```

Comment:
There's no reason to change this.

Reply:
Changed as per request here:
http://bitbucket.hq.viviport.com:7990/projects/PROV/repos/keycloak-elasticbeanstalk/pull-requests/31/overview?commentId=148947&action=view

Reply:
I made the request to change

1. To mark it as an internal build
2. To prevent future conflicts when 1.0.5 is actually out.